### ISSUE TITLE
Changed Type and location of beam only got type change

### ISSUE THREAD

**Issue by joaogdsj** [mattmas/Metamorphosis#33]
Hi,

I made two changes on a beam 

![image](https://user-images.githubusercontent.com/50149362/57067731-bb68a580-6cc7-11e9-80c1-c1ec8cec7ee9.png)

I changed its position and its type and I only got the type change on the report plus the deleted type that no longer exits in my module.

![image](https://user-images.githubusercontent.com/50149362/57067822-fb2f8d00-6cc7-11e9-88d1-737a3ba3cd33.png)

Could i get both changes of the element?

**Comment by mattmas** [comment]
Our feeling was that if we showed all of the change types, it would double or triple the number of changes shown (for example, most type changes would trigger a change in area/volume/mass or other parameters - and the type parameter itself). All in all, we figured it was better to pick the "primary" change and not flood the users with 3x of the changes.  (but your point about two separate and discrete changes is good - we were more worried about one change in Revit trigger two or three reported changes in our tool).
